benchmark/fio_benchmark_cleanup.yml

```diff
@@ -5,54 +5,13 @@
     _pool_name: "{{ fiobench_pool_name | default('fiobench') }}"
     _ceph_args: "--id {{ fiobench_client_name | default('fiobench') }}"
   tasks:
-    - name: Get bench_direct_device_name
```

Comment:
I do not think we want to remove this logic.  We ant to make it dynamic so that we only use this logic if we are also running a benchmark that will take advantage of it.  The EBS benchmarks defined in group_vars/benchmark_hosts/00-defaults.yml will require this block